modules/ROOT/pages/flow-component.adoc

```diff
@@ -45,13 +45,11 @@ up business events. You can also set up error handling for a component
 
 If you set the initial state to Stopped (`initialState="stopped"`), you need to use xref:runtime-manager::flow-management.adoc[Runtime Manager] to activate the flow, or you can simply reset the Flow configuration to `Started`. Note that the console
 output for this state looks something like this: `Flow flow_component_ex has not been started (initial state = 'stopped')`
-| Max Concurrency (`maxConcurrency`) a| Optional. The default is limited only by the availability of threads. The container thread pool determines the maximum level of parallelism that the flow can use to optimize the performance when processing messages.
+| Max Concurrency (`maxConcurrency`) a| Optional. The default is limited only by the availability of threads. The container thread pool determines the maximum level of parallelism that the flow can use to optimize the performance when processing messages. You can use Max Concurrency to prevent the flow from accepting new messages while the configured number of maximum request is being processed.
 
-Example: `1` (XML example: `maxConcurrency="1"`) .
+Set `maxConcurrency="1"` to force the flow to process requests sequentially, one at a time.
 
-You might set it to `1` to produce the same behavior as the synchronous processing strategy in Mule 3. You can also use it to stop accepting new work while the given number of requests are getting processed. Note that in Mule 3, some people use notifications and error handlers to stop the flow at some point and then restart it later, but in Mule 4, you can produce this same effect by using Max Concurrency.
-
-Refer to the documentation of xref:execution-engine.adoc#backpressure[Back-pressure] for details on how this is handled.
+See xref:execution-engine.adoc#backpressure[Back-pressure] for details about how Mule handles reaching Max Concurrency limits.
```

Comment:
For details about what? I don't understand what "how Mule handles reaching Max Concurrency limits" means.
Do you mean "what Mule does when the value of Max Concurrency is reached at a moment in time, yet message continue to be sent to the app"?